src/Mod/Draft/draftviewproviders/view_wire.py

```diff
@@ -114,14 +125,18 @@ def updateData(self, obj, prop):
         return
 
     def onChanged(self, vobj, prop):
-        if prop in ["EndArrow","ArrowSize","ArrowType","Visibility"]:
+        if prop in ["EndArrow","ArrowSize","ArrowTypeStart","ArrowTypeEnd","Visibility"]:
```

Comment:
Yes, of course this property would need to be handled by the `migrate_properties_0v22` function.

Reply:
Ok!

I think I'll implement this as for `migrate_properties_0v19`, therefor adding one method to handle the wires and another method to handle the dimensions.

I'm thinking that way simply because I see that now there are two `migrate_properties_0v19` methods, one in `patharray.py` and the other in `pointarray.py` but if you prefer I implement the new method in a different way just let me know!